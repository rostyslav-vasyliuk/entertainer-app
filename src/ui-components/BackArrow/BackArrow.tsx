import React from 'react';
import { MaterialIcons } from '@expo/vector-icons';
import { TEXT_COLOR } from '../../constants/color-constants';


const BackArrow = () => (
  <MaterialIcons name='arrow-back' size={24} color={TEXT_COLOR} style={{ paddingLeft: 4 }} />
);

export default BackArrow;

