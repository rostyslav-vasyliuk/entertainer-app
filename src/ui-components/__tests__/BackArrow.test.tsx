import React from 'react';
import BackArrow from '../BackArrow/BackArrow';
import renderer from 'react-test-renderer';

describe('<BackArrow />', () => {
  it('has null child', () => {
    const tree = renderer.create(<BackArrow />).toJSON();
    expect(tree.children).toBe(null);
  });

  it('rendered', () => {
    const tree = renderer.create(<BackArrow />).toJSON();
    expect(tree.children).toBeFalsy();
  });

  it('to match snapshot', () => {
    const tree = renderer.create(<BackArrow />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
