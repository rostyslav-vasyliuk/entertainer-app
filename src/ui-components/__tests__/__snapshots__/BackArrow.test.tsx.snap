// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BackArrow /> to match snapshot 1`] = `<Text />`;
